test_runner/batch_others/test_wal_acceptor.py

```diff
@@ -39,9 +38,9 @@ def test_normal_work(zenith_env_builder: ZenithEnvBuilder):
 
 
 @dataclass
-class BranchMetrics:
-    name: str
-    latest_valid_lsn: int
+class TimelineMetrics:
+    timeline_id: str
+    last_record_lsn: int
```

Comment:
Is it correct that we use last_record_lsn instead of disk_consistent_lsn here?

Reply:
Hm after some archeology I think it is correct, but the naming is old, last_record_lsn is a replacement for latest_valid_lsn, so you can safely rename old occurrences (or leave as is and I'll clean it up)

Reply:
There was only one `latest_valid_lsn` in the comment, I've replaced it.